templates/dashboard/includes/_collection_availability.html

```diff
@@ -1,20 +1,16 @@
 {% load i18n %}
 
   <div class="label label-{{ label_cls }}">
-    {% if is_available %}
+    {% if is_visible %}
       {% trans "Published" context "Collection availability status" %}
     {% elif not collection.is_published %}
     <div class="label">
       {% trans "Hidden" context "Collection availability status" %}
     </div>
-    {% elif collection.published_at %}
+    {% elif collection.is_published and collection.published_date is not None %}
```

Comment:
`collection.is_published` is `True`, no need to check twice.

The logic is still wrong though, you want to check `date is not None` first, then have a `else` block.